frontend/src/pages/project/Settings/Danger.vue

```diff
@@ -46,11 +46,11 @@
             </div>
         </div>
 
-        <FormHeading>Export Project</FormHeading>
+        <FormHeading>Copy Project</FormHeading>
         <!-- Hiding for now (0.5) -->
```

Comment:
Not sure what this is, or what needs to happen with it?

Reply:
This is to move a project out of FlowForge (exports flows and creds). Given we now have projects not sure we need this at the moment.